references.bib

```diff
@@ -1,1167 +1,1164 @@
-
-@InProceedings{entropist,
-  author = {Paul Syverson},
-  title = {Why {I'm} not an Entropist},
-  booktitle = {Proceedings of Security Protocols XVII: 17th International Workshop, April 2009, Revised Selected Papers},
-  pages = {231--239},
-  editor = {Bruce Christianson and James A. Malcolm and Vashek Maty\'{a}\v{s} and Michael Roe},
-  year = 2013,
-  www_pdf_url = "http://www.syverson.org/entropist-final.pdf",
-  publisher = {Springer-Verlag, LNCS 7028},
-  www_tags={selected},
-}
+%% This BibTeX bibliography file was created using BibDesk.
+%% http://bibdesk.sourceforge.net/
+
+
+%% Created for Aggelos Kiayias at 2019-01-29 02:25:38 +0100 
+
+
+%% Saved with string encoding Unicode (UTF-8) 
+
+
+
+@inproceedings{DBLP:conf/sp/Das0MK18,
+	Author = {Debajyoti Das and Sebastian Meiser and Esfandiar Mohammadi and Aniket Kate},
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/sp/Das0MK18},
+	Booktitle = {2018 {IEEE} Symposium on Security and Privacy, {SP} 2018, Proceedings, 21-23 May 2018, San Francisco, California, {USA}},
+	Crossref = {DBLP:conf/sp/2018},
+	Date-Added = {2019-01-29 02:25:00 +0100},
+	Date-Modified = {2019-01-29 02:25:00 +0100},
+	Doi = {10.1109/SP.2018.00011},
+	Pages = {108--126},
+	Timestamp = {Fri, 24 Aug 2018 00:34:59 +0200},
+	Title = {Anonymity Trilemma: Strong Anonymity, Low Bandwidth Overhead, Low Latency - Choose Two},
+	Url = {https://doi.org/10.1109/SP.2018.00011},
+	Year = {2018},
+	Bdsk-Url-1 = {https://doi.org/10.1109/SP.2018.00011}}
+
+@proceedings{DBLP:conf/sp/2018,
+	Bibsource = {dblp computer science bibliography, https://dblp.org},
+	Biburl = {https://dblp.org/rec/bib/conf/sp/2018},
+	Date-Added = {2019-01-29 02:25:00 +0100},
+	Date-Modified = {2019-01-29 02:25:00 +0100},
+	Publisher = {{IEEE}},
+	Timestamp = {Thu, 09 Aug 2018 08:55:11 +0200},
+	Title = {2018 {IEEE} Symposium on Security and Privacy, {SP} 2018, Proceedings, 21-23 May 2018, San Francisco, California, {USA}},
```

Comment:
can you remove the conference entry, as it is less useful that the more specific pointer you provide above the the individual paper within it.

Reply:
Isn't the crossref line kinda a dependency for some bib styles?  I know DBLP always works this way for whatever reason.  Anyways people could always find the proceedings from DBLP using the crossref, so removing still sounds fine.